frontend/src/pages/distributedWorkloads/GlobalDistributedWorkloadsRoutes.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { Navigate, Route } from 'react-router-dom';
+import ProjectsRoutes from '~/concepts/projects/ProjectsRoutes';
+
+import { useDistributedWorkloadsTabs } from '~/pages/distributedWorkloads/global/useDistributedWorkloadsTabs';
+import GlobalDistributedWorkloads from '~/pages/distributedWorkloads/global/GlobalDistributedWorkloads';
+
+const GlobalDistributedWorkloadsRoutes: React.FC = () => {
+  const tabs = useDistributedWorkloadsTabs();
+  const firstAvailableTab = tabs.find((tab) => tab.isAvailable);
+
+  return (
+    <ProjectsRoutes>
+      <Route index element={<Navigate to={firstAvailableTab?.path || '..'} replace />} />
```

Comment:
404 if you have no available tabs

Reply:
Don't assume the history or navigation hierarchy -- you'll end up with bugs if we shift things outside of this. Just show 404 if they end up on this path.

Reply:
Makes sense. Ok.

Reply:
On this thought, I wonder if I should have some `const distributedWorkloadsBasePath = '/distributedWorkloads'` somewhere, there are various places I hard-code that for navigation (that's how it's done elsewhere)

Reply:
You're on a route -- you want to pick a tab -- you typically do `navigate route="."` to catch missing paths and send them to the "home" of your sub-route.

But you want a tab id, your code is saying "I don't have a tab id and no tabs are available" -- that's a 404 Route... This is no different than if they typed `/asdfasdfasdf` -- that's a 404 route.

Reply:
Yeah I got ya, I implemented the 404. Sorry, this is a different thought. I just meant there are places in the code where the base path for the whole page is hard-coded, for example:

```tsx
    <ProjectSelectorNavigator
      getRedirectPath={(namespace) => `/distributedWorkloads/${tabConfig.path}/${namespace}`}
    />
```

so I was thinking maybe the `/distributedWorkloads` should be a constant somewhere in case we want to change that path in the future without updating code in several places. but today the same problem exists for e.g. `/modelServing`.

Reply:
You speak of https://issues.redhat.com/browse/RHOAIENG-2455 -- I would like to do this, but we should have a proper execution to it.

Reply:
aha! good stuff.